Automattic/wp-calypso
# Switch to separate flags for wooexpress medium and small

Related to #74123

## Proposed Changes
Add `plans/wooexpress-medium` and `plans/wooexpress-small` (only enabled on dev).

## Testing Instructions

After switching to this branch, run yarn feature-search wooexpress.

The output should look like this:

![image](https://user-images.githubusercontent.com/917632/223199341-1401e1b4-2f46-4b2f-bb54-01486d237105.png)





*

## Pre-merge Checklist
